rpc/src/slot_status_notifier.rs

```diff
@@ -0,0 +1,23 @@
+use {
```

Comment:
It feels a little awkward having this file under RPC. This is strictly used for passing information to Geyser plugins, right? Do we actually leverage this for RPC in any way?

Reply:
We are utilizing RpcCompletedSlotsService to notify the completed status -- I would like to avoid creating another service for the same purpose -- it would be not as efficient. The geyser plugin framework depends on rpc -- that is why we have to elevate the def of the interface to RPC. Maybe the "proper" way is to abstract out  RpcCompletedSlotsService  to its own crate to be shared by both rpc and geyser -- with more changes. It is not first time we did this though, OptimisticallyConfirmedBankTracker was used the same way to notify other slots status.